.pipelines/templates/template-push-images-to-acr-tagged.yml

```diff
@@ -0,0 +1,14 @@
+parameters:
+  rpImageACR: ''
+steps:
+- script: |
+    set -e
+    trap 'set +e; for c in $(docker ps -aq); do docker rm -f $c; done; docker image prune -af ; rm -rf ~/.docker/config.json' EXIT
+
+    export RP_IMAGE_ACR=${{ parameters.rpImageACR }}
+
+    az acr login --name "$RP_IMAGE_ACR"
+    # azure checkouts commit, so removing master reference when publishing image
+    export BRANCH=$(Build.SourceBranchName)
+    make publish-image-aro-tag
```

Comment:
Copies the original pipeline, this is the only difference. Call tagged makefile entrypoint.